python/lib/bidsreader.py

```diff
@@ -19,12 +18,12 @@
         exit(lib.exitcode.INVALID_IMPORT)
 
 # BIDSLayoutIndexer is required for PyBIDS >= 0.12.1
-bids_pack_version = list(map(int, bids.__version__.split('.')))
-if (bids_pack_version[0] > 0
-        or bids_pack_version[1] > 12
-        or (bids_pack_version[1] == 12 and bids_pack_version[2] > 0)):
-    
-	from bids import BIDSLayoutIndexer
+# bids_pack_version = list(map(int, bids.__version__.split('.')))
+# if (bids_pack_version[0] > 0
+#     or bids_pack_version[1] > 12
+#    or (bids_pack_version[1] == 12 and bids_pack_version[2] > 0)):
+
+# 	from bids import BIDSLayoutIndexer
```

Comment:
shouldn't this be removed instead of commented out? It seems to be one in the `load_bids_data` function now

Reply:
It will be probably uncommented whenever I spent time on this